ruyi/cli/completion.py

```diff
@@ -0,0 +1,75 @@
+"""
+helper functions for CLI completions
+"""
+
+from typing import cast, Callable, Optional, Protocol, Sequence, Any, TYPE_CHECKING
+import argparse
```

Comment:
Imports should generally be sorted alphabetically; the only exceptions are those with side-effects thus requiring a certain import order, but such cases should always get documented with comments, so it is not the case here and we should maintain alphabetical order.

```suggestion
import argparse
from typing import cast, Callable, Optional, Protocol, Sequence, Any, TYPE_CHECKING
```